src/main/java/com/mosaicchurchaustin/oms/data/constants/MosaicAuthority.java

```diff
@@ -0,0 +1,25 @@
+package com.mosaicchurchaustin.oms.data.constants;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum MosaicAuthority {
+    ADMIN("admin");
```

Comment:
Are there different levels of admin to consider?  🤷

Reply:
There may be, but think of this one as a minimal requirement - ie, to access ANY /admin url, you need admin authority.  To do specific admin functionality (ie, add a user vs modifying an item), you may need more specific permissions which we can add later.  But having this allows a broad restriction to admin endpoints.